src/Discord.ts

```diff
@@ -195,19 +208,32 @@ export class DiscordSDK implements IDiscordSDK {
     }
   }
 
+  private parseMajorMobileVersion(): number | null {
+    if (this.mobileAppVersion && this.mobileAppVersion.includes('.')) {
+      try {
+        return parseInt(this.mobileAppVersion.split('.')[0]);
+      } catch {
+        return null;
+      }
+    }
+    return null;
```

Comment:
Nit: Might be cool if we could return `UNKNOWN` or some other constant, so we could determine if the version is absent because of an error or because the version is too old to attempt to provide it